Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/valueset/internal/DccValueSetMapper.kt

```diff
@@ -9,11 +9,39 @@ import timber.log.Timber
 import java.util.Locale
 
 internal fun ValueSetsOuterClass.ValueSets.toValueSetsContainer(languageCode: Locale): ValueSetsContainer {
-    Timber.d("toValueSetsContainer(valueSets=%s, languageCode=%s)", this, languageCode)
     return ValueSetsContainer(
         vaccinationValueSets = toVaccinationValueSets(languageCode = languageCode),
         testCertificateValueSets = toTestCertificateValueSets(languageCode = languageCode)
-    ).also { Timber.tag(TAG).d("Created %s", it) }
+    ).also {
+        Timber.tag(TAG).d(
+            """
+        VaccinationValueSets {
+            languageCode = %s
+            tg.size = %d
+            vp.size = %d
+            mp.size = %d
+            ma.size = %d
+        }
+        TestCertificateValueSets {
+            languageCode = %s
+            tg.size = %d
+            tt.size = %d
+            ma.size = %d
+            tr.size = %d
+        }
+            """.trimIndent(),
+            languageCode,
+            it.vaccinationValueSets.tg.items.size,
+            it.vaccinationValueSets.vp.items.size,
+            it.vaccinationValueSets.mp.items.size,
+            it.vaccinationValueSets.ma.items.size,
+            languageCode,
+            it.testCertificateValueSets.tg.items.size,
+            it.testCertificateValueSets.tt.items.size,
+            it.testCertificateValueSets.ma.items.size,
+            it.testCertificateValueSets.tr.items.size,
+        )
+    }
```

Comment:
I think just simple message would be just fine ?

Reply:
yes, it could be enough, updated